plexapi/video.py

```diff
@@ -709,7 +709,7 @@ def _defaultSyncTitle(self):
 
 
 @utils.registerPlexObject
-class Episode(Video, Playable, ArtMixin, PosterMixin, RatingMixin,
+class Episode(Video, Playable, ArtMixin, PosterMixin, RatingMixin, LabelMixin,
         CollectionMixin, DirectorMixin, WriterMixin):
```

Comment:
Alphabetical `LabelMixin` goes after `DirectorMixin`.

Reply:
The list already not alphabetical...

54731dd,  f3b6ad2